### PR TITLE
MGMT-5295 - Use AI configuration to configure DiscoveryISO and RHCOS …

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…with mirror registries

the values for mirror registry CA and mirror registry conf file will be set into Assisted Service during deployment
the configuration of the Discovery ISO and RCHOS will be based on those value ( in case they are present)

### CODE REVIEW

File: internal/ignition/ignition.go [hunk 1]
```diff
@@ -1275,10 +1278,12 @@ func (ib *ignitionBuilder) FormatDiscoveryIgnitionFile(cluster *common.Cluster,
 		ignitionParams["ServiceIPs"] = dataurl.EncodeBytes([]byte(GetServiceIPHostnames(cfg.ServiceIPs)))
 	}
 
-	if cluster.ImageInfo.MirrorRegistriesConfig != "" {
-		ignitionParams["MirrorRegistriesConfig"] = base64.StdEncoding.EncodeToString([]byte(cluster.ImageInfo.MirrorRegistriesConfig))
-		ignitionParams["MirrorRegistriesCAConfig"] = base64.StdEncoding.EncodeToString([]byte(cluster.ImageInfo.CaConfig))
-	}
+	/*
```
Comment (tsorya) [inline internal/ignition/ignition.go:1281]: remove?

Comment (yevgeny-shnaidman) [reply internal/ignition/ignition.go]: fixed

**Pull request merged**